query_and_submit.py

```diff
@@ -18,52 +18,22 @@
 interval_start = one_hour_ago.strftime("%Y-%m-%dT%H:00:00Z")
 interval_end = one_hour_ago.strftime("%Y-%m-%dT%H:59:59Z")
 
-count_metrics = {}
-metrics = []
+metrics = {}
 
 for metric_name, query in config["queries"].items():
-  count_sql = query.get("countSql")
-  content_sql = query.get("contentSql")
+  sql = query["countSql"]
+  condition_keyword = "and" if re.search("where", sql, re.IGNORECASE) else "where"
   timestamp_col = query["timestampColumn"]
-  
-  if content_sql:
-    condition_keyword = "and" if re.search("where", content_sql, re.IGNORECASE) else "where"
-    db_result = subprocess.check_output(["psql", os.environ["DBURI"], "-Atc", f"{content_sql} {condition_keyword} \"{timestamp_col}\" between '{interval_start}' and '{interval_end}'"]).decode().split("\n")
-    db_result = filter(lambda single_result: single_result != "", db_result)
+  metrics[metric_name] = json.loads(subprocess.check_output(["psql", os.environ["DBURI"], "-Atc", f"{sql} {condition_keyword} \"{timestamp_col}\" between '{interval_start}' and '{interval_end}'"]))
 
-    content_metric = {}
-
-    for result in db_result:
-      category, content, created_at = result.split("|")
-
-      content_metric["category"] = category
-      content_metric["content"] = content
-      content_metric["date"] = created_at
-
-      metrics.append({
-        **content_metric
-      })
-
-  else:
-    condition_keyword = "and" if re.search("where", count_sql, re.IGNORECASE) else "where"
-    count_metrics[metric_name] = json.loads(subprocess.check_output(["psql", os.environ["DBURI"], "-Atc", f"{count_sql} {condition_keyword} \"{timestamp_col}\" between '{interval_start}' and '{interval_end}'"]))
-
-if count_metrics:
-  metrics.append({
+requests.post(
```

Comment:
I am wondering whether we want to keep the response check: 
```suggestion
response = requests.post(
```

Reply:
good idea 👏